docs/csharp/distinguish-delegates-events.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Distinguising Delegates and Events | Microsoft Docs
```

Comment:
Can you also correct the title (not only in metadata, but on line 15) -- Distinguishing --> Distinguishing